bee/build.gradle

```diff
@@ -3,11 +3,11 @@ apply plugin: 'com.neenbedankt.android-apt' // Used by butterknife
 
 // Default parameters to empty string.
 if (!project.hasProperty("debug_sdk_key")) {
-    ext.debug_sdk_key = ""
+    ext.debug_sdk_key = debug_sdk_key
```

Comment:
Please keep it as it was, these lines are only affecting a default value to our configuration variable.